README.md

```diff
@@ -13,37 +13,39 @@ The objective of this project is to utilize social network analysis techniques t
 <img src="https://raw.githubusercontent.com/wiki/prasadtalasila/IRCLogParser/sample_images/infomap_CU.png" width="400px"></img>
 
 
-## Usage
-
-The library's working has been modularised into many modules namely input, analysis, output, visualisation and validation. The [sample.py file](./sample.py) very comprehensively presents how one can use IRCLogParser for parsing and analysis.
-
-A page on typical workflow for using IRCLogParser also exists on the documentation [here](http://prasadtalasila.github.io/IRCLogParser/TypicalUsage.html).
-
-Some of the visualisable sample outputs can be on the wiki, giving an idea about IRCLogParser's capabilites.
-
-
 ## Documentation
 
-IRCLogParser uses [Sphinx Python Documentation Genertor](http://www.sphinx-doc.org/en/stable/) for generating documentation of the library. The documentation is setup to work with [Google Style Docstrings](http://www.sphinx-doc.org/en/stable/ext/example_google.html) which eases the documentation writing process.
+IRCLogParser uses [Sphinx Python Documentation Generator](http://www.sphinx-doc.org/en/stable/) for generating documentation of the library.
+The documentation is auto-generated from the commits made on the [dev](https://github.com/prasadtalasila/IRCLogParser/tree/dev) branch. The documentation corresponding to the latest commit on dev branch is hosted here.
 
 The documentation is deployed on the branch `gh-pages` who's updation has been made automatic by the a [bash script](./ext/docs_auto_deploy.sh) which otherwise requires to run `make html` in docs directory after every commit.
 
 You can view the documentation hosted on `gh-pages` [here](http://prasadtalasila.github.io/IRCLogParser/).
 
 
+## Installation
+
+IRCLogParser depends on various third-party libraries which are handled by [setup.py](./setup.py). 
+Run the "install.sh" script in the root directory to install these dependencies and other OS level
+dependencies for the third-party libraries.
+
+
 ## Testing
 
 Presently, IRCLogParser has various end-to-end implemented which reside in the [test directory](./test/).
-To run the tests locally on your machine, run `python -m unittest -v tests` in the test directory.
+To run the all the tests locally on your machine, run `python -m unittest -v discover -s test/` in the root directory.
```

Comment:
In the next two lines, separate the commands into code snippets.

Ex: To run the all the tests locally on your machine, run
```bash
python -m unittest -v discover -s test/
```
Placing the commands separately reduces errors in copy-paste process.